packages/core/src/state/reducer.ts

```diff
@@ -80,15 +87,28 @@ export default function reducer(
           ...state.results,
           [action.meta.key]: result,
         };
-        results = applyUpdatersToResults(results, result, action.meta.updaters);
-        if (action.meta.update) {
-          const updaters = action.meta.update(
+        try {
```

Comment:
the diff is weird here, this is just an indentation due to adding `try {`

Reply:
looks better when whitespace changes are hidden, at least

Reply:
hmm, are you not using the website? is the github desktop or some other method?

Reply:
The website has a “hide white space” node, under the gear icon (since a couple years ago). You can also append ?w=1 to any diff url (since always)